### PR TITLE
move inactive approvers to emeritus_approvers

### PR DESCRIPTION
**Pull request by andrewsykim**
Signed-off-by: Andrew Sy Kim <kim.andrewsy@gmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Currently going through subproject OWNER files under SIG Cloud Provider and moving approvers that have been inactive for more than 1 year to emeritus status.

I'm going to time box this PR til May 15th (next Provider AWS meeting) in case any of the approvers express interest in contributing to the project. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,27 +1,20 @@
 approvers:
 - justinsb
-- zmerlynn
```
Comment (andrewsykim) [inline OWNERS:3]: cc @zmerlynn https://github.com/kubernetes/cloud-provider-aws/commits?author=zmerlynn

File: OWNERS [hunk 2]
```diff
@@ -1,27 +1,20 @@
...
-- gnufied
```
Comment (andrewsykim) [inline OWNERS:4]: cc @gnufied https://github.com/kubernetes/cloud-provider-aws/commits?author=gnufied

File: OWNERS [hunk 3]
```diff
@@ -1,27 +1,20 @@
...
-- jsafrane
```
Comment (andrewsykim) [inline OWNERS:5]: cc @jsafrane https://github.com/kubernetes/cloud-provider-aws/commits?author=jsafrane

File: OWNERS [hunk 4]
```diff
@@ -1,27 +1,20 @@
...
 - nckturner
-- micahhausler
```
Comment (andrewsykim) [inline OWNERS:7]: cc @micahhausler https://github.com/kubernetes/cloud-provider-aws/commits?author=micahhausler

File: OWNERS [hunk 5]
```diff
@@ -1,27 +1,20 @@
...
-- mcrute
```
Comment (andrewsykim) [inline OWNERS:8]: cc @mcrute https://github.com/kubernetes/cloud-provider-aws/commits?author=mcrute

File: OWNERS [hunk 6]
```diff
@@ -1,27 +1,20 @@
...
-- gyuho
```
Comment (andrewsykim) [inline OWNERS:9]: cc @gyuho https://github.com/kubernetes/cloud-provider-aws/commits?author=gyuho

File: OWNERS [hunk 7]
```diff
@@ -1,27 +1,20 @@
...
-- d-nishi
```
Comment (andrewsykim) [inline OWNERS:10]: cc @d-nishi https://github.com/kubernetes/cloud-provider-aws/commits?author=d-nishi

File: OWNERS [hunk 8]
```diff
@@ -1,27 +1,20 @@
...
 - M00nF1sh
 - leakingtapan
 - andrewsykim
 reviewers:
-- gnufied
-- jsafrane
 - justinsb
-- zmerlynn
-- chrislovecnm
```
Comment (andrewsykim) [inline OWNERS:19]: cc @chrislovecnm https://github.com/kubernetes/cloud-provider-aws/commits?author=chrislovecnm

**Comment by andrewsykim** [comment]
/hold

Time boxing to May 15th (next Provider AWS meeting)

File: OWNERS (hunk 2 above)
Comment (gnufied) [reply OWNERS:4]: I would still preferably keep myself here. I am primary maintainer/author/reviewer of several features and bug fixes in AWS's storage layer. I don't usually review other cloudprovider features though.

Comment (andrewsykim) [reply OWNERS:4]: None of the storage integration are included in the CCM component, are you still interested in reviewing PRs here?

Comment (andrewsykim) [reply OWNERS:4]: Worth noting that the core functionality in the CCM is still imported from https://github.com/kubernetes/legacy-cloud-providers/tree/master/aws which you are still an approver for.

Comment (gnufied) [reply OWNERS:4]: Ack

File: OWNERS (hunk 5 above)
Comment (mcrute) [reply OWNERS:8]: I haven't had much time to dedicate to this lately so I'm okay with being removed as an active maintainer. Any reason not to just push me into emeritus instead?

Comment (andrewsykim) [reply OWNERS:8]: You're in emeritus, see line 18 (diff is rendered weirdly).

Comment (mcrute) [reply OWNERS:8]: Thanks for the cleanup! I read the diff out of context. I hope to have more time to contribute to cloud provider in the future. Until then...

/lgtm

File: OWNERS (hunk 4 above)
Comment (micahhausler) [reply OWNERS:7]: LGTM

**Comment by andrewsykim** [comment]
/hold cancel

This was discussed in last week's meeting and we agreed to merge this as folks have either agreed to the changes or have not responded.

**Pull request merged**